lib/src/parse/stylesheet.dart

```diff
@@ -1012,15 +1021,15 @@ abstract class StylesheetParser extends Parser {
     var identifiers = <String>{};
     var variables = <String>{};
     do {
-      whitespace();
+      whitespace(allowNewlines: false);
       withErrorMessage("Expected variable, mixin, or function name", () {
         if (scanner.peekChar() == $dollar) {
           variables.add(variableName());
         } else {
           identifiers.add(identifier(normalize: true));
         }
       });
-      whitespace();
+      whitespace(allowNewlines: false);
     } while (scanner.scanChar($comma));
```

Comment:
It may require restructuring the loop, but we should allow newlines after a comma since ending a statement with it is not valid.